src/Moq/ExpressionExtensions.cs

```diff
@@ -9,7 +9,7 @@
 using System.Linq.Expressions;
 using System.Reflection;
 using System.Text;
-
+using Moq.Linq;
```

Comment:
Once you've moved `ConstructorCallVisitor` out of the `Moq.Linq` namespace, please adjust this import. Also, as mentioned in the other review comment, please insert a blank line above, so that namespace imports are grouped by the main namespaces (`System.*`, `Moq.*`).